### PR TITLE
[#2216] Log rather than throw exceptions

### PR DESCRIPTION
**Pull request by justinmclean**
### What changes were proposed in this pull request?

Log rather than throw exceptions

### Why are the changes needed?

To fix a possible security issue.

Fix: #2216

### Does this PR introduce _any_ user-facing change?

No.

### How was this patch tested?

Tested locally.

### CODE REVIEW

**Comment by diqiu50** [comment]
Please remove the Irrelevant message in the description of this pr.

File: server/src/main/java/com/datastrato/gravitino/server/web/ConfigServlet.java [hunk 1]
```diff
@@ -46,19 +49,17 @@ public ConfigServlet(ServerConfig serverConfig) {
 
   @Override
   protected void doGet(HttpServletRequest req, HttpServletResponse res)
-      throws UnsupportedEncodingException, IllegalStateException, IOException {
+      throws IllegalStateException, IOException {
```
Comment (diqiu50) [inline server/src/main/java/com/datastrato/gravitino/server/web/ConfigServlet.java:52]: Why we remove the  exception of `UnsupportedEncodingException`

Comment (justinmclean) [reply server/src/main/java/com/datastrato/gravitino/server/web/ConfigServlet.java:52]: Because it's an IOException

**Review by diqiu50 — APPROVED** [review]
LGTM

**Pull request merged**